mozregression/bisector.py

```diff
@@ -245,7 +245,21 @@ def handle_merge(self):
         LOG.debug("Found commit message:\n%s\n" % msg)
         branch = find_branch_in_merge_commit(msg)
         if not (branch and len(push.changesets) >= 2):
-            return
+            # So we did not found a branch. Let's try with inbound anyway
+            if get_category(most_recent_push.repo_name) != 'integration' and \
+               len(push.changesets) >= 2:
+                jp2 = JsonPushes("mozilla-inbound")
+                try:
+                    data = jp2.pushes_within_changes(
+                        push.changesets[0]['node'],
+                        push.changesets[-1]['node'])
+                except MozRegressionError:
+                    return
+                LOG.info("************* Switching to mozilla-inbound")
```

Comment:
Could you make the log message reflect the fact that we couldn't automatically determine the branch, and we're falling back to inbound? I think that would be helpful for debugging.